syft/workers/base.py

```diff
@@ -221,6 +225,27 @@ def registration_enabled(self):
         finally:
             self.is_client_worker = True
 
+    def connect_to_mpc(self, rank, world_size):
+        import os
+        default_args = {
+             "DISTRIBUTED_BACKEND": "gloo",
+             "MASTER_IP": "127.0.0.1",
+             "MASTER_PORT": 9000,
+             "WORLD_SIZE": world_size,
+             "RANK": rank,
```

Comment:
You should also add "RENDEZVOUS" as "env://" so that init_method is set appropriately as stated in the [docs](https://pytorch.org/docs/stable/distributed.html#environment-variable-initialization)

Reply:
Currently I use the ```store```, ```init_method``` is mutually exclusive with ```store```